mod/metrics/lib/card/answer_query.rb

```diff
@@ -1,5 +1,10 @@
 class Card
   class AnswerQuery
```

Comment:
codeclimate considered this file our worst code smell, because it had so many methods.  I broke it out into several pieces.